sql/core/src/main/scala/org/apache/spark/sql/expressions/Window.scala

```diff
@@ -113,12 +113,12 @@ object Window {
    * Creates a [[WindowSpec]] with the frame boundaries defined,
    * from `start` (inclusive) to `end` (inclusive).
    *
-   * Both `start` and `end` are relative positions from the current row. For example, "0" means
+   * Both `start` and `end` are relative positions to the current row. For example, "0" means
    * "current row", while "-1" means the row before the current row, and "5" means the fifth row
    * after the current row.
    *
-   * We recommend users use `Window.unboundedPreceding`, `Window.unboundedFollowing`,
-   * and `Window.currentRow` to specify special boundary values, rather than using integral
+   * We recommend users to use [[Window.unboundedPreceding]], [[Window.unboundedFollowing]],
```

Comment:
Leaving 'that' aside is incorrect -- see http://dictionary.cambridge.org/dictionary/english/recommend where *to* is even highlighted to make the point.